KommunicateObjcSample/ViewController.m

```diff
@@ -25,7 +24,10 @@ - (void)viewDidLoad {
                     if(clientChannelKey) {
                         NSLog(@"Client channel key %@", clientChannelKey);
                         dispatch_async(dispatch_get_main_queue(), ^{
-                            [Kommunicate showConversationWithGroupId:clientChannelKey from:self completionHandler:^(BOOL shown) {
+                            [Kommunicate showConversationWithGroupId:clientChannelKey
+                                                                from:self
+                                                    prefilledMessage:nil
```

Comment:
We can omit this param as it contains a default value.

Reply:
Only this method it as an access.

Reply:
Yeah, in Obj-C we can't skip.